src/SD.cpp

```diff
@@ -167,7 +176,7 @@ bool SDClass::remove(const char *filepath)
 
 File SDClass::openRoot(void)
 {
-  File file = File(_fatFs.getRoot());
+  File file = open(_fatFs.getRoot());
```

Comment:
As `open` call `f_opendir` no need to call it after
```suggestion
  return = open(_fatFs.getRoot());
```

Reply:
I've updated your PR.